kolibri/core/auth/models.py

```diff
@@ -344,6 +344,21 @@ def infer_dataset(self, *args, **kwargs):
         )
 
 
+def validate_username(value):
+    try:
+        validators.EmailValidator()(value)
+    except ValidationError:
+        # for kolibri backwards compatibility, if the username is not an email:
+        validators.RegexValidator(
```

Comment:
See note below - this can be defined in the module scope and reused.

Reply:
Done